installing/installing_bare_metal/bare-metal-using-bare-metal-as-a-service.adoc

```diff
@@ -0,0 +1,41 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="bare-metal-using-bare-metal-as-a-service"]
+= Using bare metal as a service
+include::_attributes/common-attributes.adoc[]
+:context: bare-metal-using-bmaas
+
+toc::[]
+
+{product-title}'s Bare Metal as a Service (BMaaS) feature enables you to provision and manage bare-metal nodes by using the Metal^3^ API. These nodes, external to the {product-title} cluster, can host workloads that might not be suitable for containerisation or virtualisation, such as applications requiring direct hardware access, high-performance computing tasks or legacy applications. BMaaS provides the following capabilities:
+
+* Provisioning of bare-metal nodes, including initial configuration.
+* Lifecycle management, such as power management, firmware updates, and decommissioning.
+
+As standalone systems, these nodes operate independently of the {product-title} cluster and support diverse workloads by integrating bare-metal resources with containerized and virtualized applications. BMaaS supports {op-system-base-full}, Fedora, and CentOS on the bare-metal nodes.
```

Comment:
BMaaS has been tested with Red Hat Enterprise Linux, CentOS Stream 9 and Fedora Core 42. BMaaS is expected to be able to run other operating systems, however only the ones listed have been tested by Red Hat engineers.

Reply:
Done.